pom.xml

```diff
@@ -1189,30 +1189,15 @@
                 </exclusions>
             </dependency>
 
+            <!-- Start dependencies for querying Hudi table-->
             <dependency>
                 <groupId>org.apache.hudi</groupId>
-                <artifactId>hudi-common</artifactId>
+                <artifactId>hudi-presto-bundle</artifactId>
                 <version>${dep.hudi.version}</version>
                 <exclusions>
                     <exclusion>
-                        <groupId>org.apache.hbase</groupId>
-                        <artifactId>hbase-server</artifactId>
-                    </exclusion>
-                    <exclusion>
-                        <groupId>org.apache.orc</groupId>
-                        <artifactId>orc-core</artifactId>
-                    </exclusion>
-                    <exclusion>
-                        <groupId>org.objenesis</groupId>
-                        <artifactId>objenesis</artifactId>
-                    </exclusion>
-                    <exclusion>
-                        <groupId>commons-logging</groupId>
-                        <artifactId>commons-logging</artifactId>
-                    </exclusion>
-                    <exclusion>
-                        <groupId>org.slf4j</groupId>
-                        <artifactId>jcl-over-slf4j</artifactId>
+                        <groupId>com.esotericsoftware</groupId>
```

Comment:
Why are their still excludes if the hudi-presto-bundle is shaded ?

Reply:
Removed unnecessary exclusions. There are some hudi modules which we need to exclude and then there is protobuf, which if not excluded, was causing the build to fail due to some conflicting resource files.